### PR TITLE
Feature/applics 89 welsh exam timetable

### PR DESCRIPTION
**Pull request by emil-placheta**
https://pins-ds.atlassian.net/browse/APPLICS-89

Introduce Welsh translation to examination-timetable page

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/__tests__/unit/utils/is-before-or-after-date.test.js [hunk 1]
```diff
@@ -120,10 +128,12 @@ describe('#utils/is-before-or-after-date', () => {
...
 				beforeEach(() => {
 					jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
-					result = getDateTimeExaminationEnds(date, extensionDate, '2022-09-14');
+					result = getDateTimeExaminationEnds(date, extensionDate, '2022-09-14', i18n);
 				});
 				it('should return the correct before date sentence. ', () => {
-					expect(result).toEqual('The examination is expected to close on 19 August 2022');
+					expect(result).toEqual(
+						'The deadline for the close of the Examination has been extended to 22 April 2023'
+					);
```
Comment (emil-placheta) [inline packages/forms-web-app/__tests__/unit/utils/is-before-or-after-date.test.js:136]: bug fix: test meant to check extension date, not the close date

File: packages/forms-web-app/src/utils/is-before-or-after-date.js [hunk 2]
```diff
@@ -25,36 +25,36 @@ const isBeforeTodayUTC = (date) => {
...
+		? `${i18n.t('examinationTimetable.examinationClosesOn')} ${formatDate(date)}` //The examination is expected to close on
+		: `${i18n.t('examinationTimetable.examinationClosedOn')} ${formatDate(date)}`; //The examination closed on
 
-const getDateTimeExaminationEnds = (closeDate, extensionCloseDate, startDate) => {
-	if (!isInvalidDate(closeDate)) return isBeforeOrAfterSentence(closeDate);
+const getDateTimeExaminationEnds = (closeDate, extensionCloseDate, startDate, i18n) => {
+	if (!isInvalidDate(extensionCloseDate)) return handleGrantedExtension(extensionCloseDate, i18n);
 
-	if (!isInvalidDate(extensionCloseDate)) return handleGrantedExtension(extensionCloseDate);
+	if (!isInvalidDate(closeDate)) return isBeforeOrAfterSentence(closeDate, i18n);
```
Comment (emil-placheta) [inline packages/forms-web-app/src/utils/is-before-or-after-date.js:53]: bug fix: changed order of `closeDate` and `ExtensionCloseDate` as if the `closeDate` was provided, the extension would never get chance to return, even if it was there and valid

**Review by emil-placheta — COMMENTED** [review]
This PR has a couple of small bug fixes alongside the translation changes, added notes in relevant place for clarity

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**